### PR TITLE
Feature/delete acronym request after denied

### PR DESCRIPTION
**Pull request by ne-cabrerav**
At the 3rd denial, the acronym is deleted

### CODE REVIEW

File: lambda/add_meaning.py [hunk 1]
```diff
@@ -255,16 +255,23 @@ def persistDecision(acronym, userId, decision):
...
+            Key={
             'Acronym': acronym
-        },
-        UpdateExpression=f"set {decisionStr}=:d",
-        ExpressionAttributeValues={
-            ':d': reviewers,
-        },
-        ReturnValues="UPDATED_NEW"
-    )
+            }
```
Comment (oijusti) [inline lambda/add_meaning.py:262]: Formatting

File: lambda/add_meaning.py [hunk 2]
```diff
@@ -255,16 +255,23 @@ def persistDecision(acronym, userId, decision):
     
     reviewers.append(userId)
 
-    response = table.update_item(
-        Key={
+    if not decision and len(reviewers) >= 3:
+        response = table.delete_item(
```
Comment (oijusti) [inline lambda/add_meaning.py:259]: I wonder what happens when an approver tries to approve or deny an acronym that was deleted from the table.

Comment (ne-cabrerav) [reply lambda/add_meaning.py:259]: It will fail with a 500 error, I will updated to return a 404

**Review by lapv1992 — APPROVED** [review]
Resolve comments

File: lambda/add_meaning.py (hunk 1 above)
Comment (oijusti) [reply lambda/add_meaning.py:266]: minor - indentation for `'Acronym': acronym`

**Review by oijusti — APPROVED** [review]

**Pull request merged**